install/ui/Makefile.am

```diff
@@ -34,3 +34,6 @@ MAINTAINERCLEANFILES =                  \
         *~                              \
         Makefile.in
 	$(NULL)
+
+install-data-hook:
+	$(INSTALL) -d -D -m 755 $(DESTDIR)$(appdir)/js/plugins
```

Comment:
make install cannot install files as root (or any other user) because make intall is typically called as unprivileged user by the rpmbuild.